corehq/apps/export/utils.py

```diff
@@ -608,7 +609,8 @@ def migrate_domain(domain, dryrun=False, force_convert_columns=False):
                 )
             except Exception as e:
                 print('Failed parsing {}: {}'.format(old_export['_id'], e))
-                raise e
+                exc_type, exc_value, exc_traceback = sys.exc_info()
+                raise exc_type, exc_value, exc_traceback
```

Comment:
thanks that does work. updated: https://github.com/dimagi/commcare-hq/pull/15092/commits/f333c776c7436174899aac6477937860eb58fd71